engine-nextgen/src/main/kotlin/graphql/nadel/enginekt/transform/query/NadelFieldToService.kt

```diff
@@ -91,7 +92,7 @@ internal class NadelFieldToService(
     }
 
     private fun isNamespacedField(field: ExecutableNormalizedField): Boolean {
-        return isNamespacedField(field, overallExecutionBlueprint.schema)
+        return isNamespacedField(field, querySchema)
```

Comment:
I think we should swap this back to engine schema. If we stick to the rule that

1. Query schema for anything user facing (validation, introspection).
2. Engine schema for almost everything else.

It's a lot easier to understand what to use. We should add a comment about that on `querySchema` somewhere too.